frontend/src/index.html

```diff
@@ -200,6 +204,14 @@ <h3 class="flex-grow break-words font-semibold" title="{{title}}">{{#helpers.hig
                     </a>
                   </div>
                 </div>
+                <!--Topics discussed in the recording-->
+                {{#_highlightResult.topics}}
```

Comment:
Having different condition for displaying the topics in comparison to displaying the More link may lead to inconsistency. Let's use `#topicsReady` here also.